src/backupNg/backupNg.spec.js

```diff
@@ -377,4 +377,193 @@ describe("backupNg", () => {
     });
     expect(vmTask.data.id).toBe(vmId);
   });
+
+  test("execute three times a rolling snapshot, a delta backup and a CR, revert to an old state & restore the backups", async () => {
+    jest.setTimeout(8e4);
+    const vmId = await xo.createTempVm({
+      name_label: "XO Test Temp",
+      name_description: "Creating a temporary vm",
+      template: config.xoTestTemplateId,
+      VDIs: [
+        {
+          size: 1,
+          SR: config.srs.defaultSr,
+          type: "user",
+        },
+      ],
+    });
+
+    const scheduleTempId = randomId();
+    const { id: jobId } = await xo.createTempBackupNgJob({
+      ...defaultBackupNg,
+      mode: "delta",
+      vms: {
+        id: vmId,
+      },
+      remotes: {
+        id: {
+          __or: [config.remotes.defaultRemote1, config.remotes.defaultRemote2],
+        },
+      },
+      schedules: {
+        [scheduleTempId]: DEFAULT_SCHEDULE,
+      },
+      settings: {
+        "": {
+          reportWhen: "never",
+          fullInterval: 3,
+        },
+        [config.remotes.defaultRemote1]: {
+          deleteFirst: true,
+        },
+        [config.remotes.defaultRemote2]: {
+          deleteFirst: false,
+        },
+        [scheduleTempId]: {
+          snapshotRetention: 2,
+          exportRetention: 2,
+          copyRetention: 2,
+        },
+      },
+      srs: {
+        id: {
+          __or: [config.srs.srLocalStorage],
```

Comment:
Why local storage?

Reply:
You are right, it is more adapted to call this : `localSr`. :+1:

Reply:
Why not a shared SR?

Reply:
I don't think we should test the CR on a shared and a local SR, but we need these two for the restoration.

Reply:
Can we let it configurable? In case if we want to test a shared and a local SR?

Reply:
You can ignore the above comment. It can have different behaviour on doing a CR on a local SR and a shared SR. Let's include them to the test.